app/locations/service.py

```diff
@@ -16,11 +19,48 @@ def poi_proposal_collection():
     return firestore_db().collection(POI_PROPOSAL_COLLECTION)
 
 
-def list_POI() -> List[POI]:
+def list_POI(
+    user_location: Tuple[float, float],
+    clazz: Optional[POIClassification] = None,

```

Comment:
I know class is a reserved keyword, but it might be better to use "classification" instead here

Reply:
Updated to `classification`.